src/main/kotlin/compiler/parser/Token.kt

```diff
@@ -53,13 +53,11 @@ enum class TokenType(val literal: String, val isKeyword: Boolean = false) {
     T_OBJREF("#xxxxx"),
 
     // Keywords
-    T_AND("and", true),
```

Comment:
It turned out I had "and/or" and "&&/||" both tokenized, but I didn't see the point of supporting both, so I decided to go with &&/|| only.